### ISSUE TITLE
[Bug Report] Autoplaylist: Playlist only picks one song out of list

### ISSUE THREAD

**Issue by redblade7** [jagrosh/MusicBot#261]
**Describe the bug**
Since version 0.2.4, bots using an autoplaylist on startup will add only one song from that playlist. This is regardless of whether the text file consists of playlists or links. If the playlist is in shuffle mode, a random song will be added; if not, only the first song in the text file will be added. If the text file consists of playlist links and is in shuffle mode, a random song from the first playlist link will be added; if not, the first song from the first playlist link will be added.

Manually adding the same playlist file after startup does not have this bug and works as expected.

**To Reproduce**
1. Add an autoplaylist to the bot.
2. Shut down the bot
3. Start up the bot
4. Only one song is added to the list (see description above). If Repeat Mode is On, that song is looped repeatedly.

**Expected behavior**
All songs on the autoplaylist should be added, in a random order if shuffle is enabled.

**Version Info (please complete the following information):**
 - Operating System: VPS running Slackware 14.2 64-bit
 - JMusicBot Version:0.2.4

**Comment by jagrosh** [comment]
Can you screenshot your bot's `settings` command output, and also paste the entire contents of the playlist file?

**Comment by redblade7** [comment]
![outrun-duncan](https://user-images.githubusercontent.com/15269155/62973078-edc88980-be04-11e9-8177-5638f16fc717.png)
[prankcalls.txt](https://github.com/jagrosh/MusicBot/files/3498641/prankcalls.txt)
[retrowave.txt](https://github.com/jagrosh/MusicBot/files/3498642/retrowave.txt)

Here are two examples.

**Comment by daranto** [comment]
Hello,
I have the same bug. Im using version: 0.2.6

**Comment by jagrosh** [comment]
If you turn repeat mode off, do other songs from the playlist play?

**Comment by daranto** [comment]
Yeah when I disable the replay function the autoplaylist is working. But does the playlist play unlimited when the replay function is not activated?

**Comment by jagrosh** [comment]
Correct, you do not need repeat mode turned on; the autoplaylist will continue to repeat. Repeat mode is primarily for songs that are added by users.
